packages/components/src/select-control/styles/select-control-styles.ts

```diff
@@ -137,6 +137,9 @@ export const Select = styled.select< SelectProps >`
 		width: 100%;
 		max-width: none;
 		cursor: pointer;
+		overflow: hidden;
```

Comment:
Is there a reason why we're using raw CSS instead of  the `Truncate` component? (I'm ok with both, just curious)

Reply:
I don't think there is a placement that works in this case with the `<select>`. The main reason though is for encapsulating the styles within the styled Select export so it can be reused in CustomSelectControl.